public/app/features/dashlinks/editor.html

```diff
@@ -65,6 +65,7 @@ <h5 class="section-heading">Links and Dash Navigation</h5>
 				<div class="gf-form">
 					<span class="gf-form-label width-6">Include</span>
 					<editor-checkbox text="Time range" model="link.keepTime" change="updated()"></editor-checkbox>
+          <editor-checkbox ng-show="link.keepTime" text="In Kibana format" model="link.kibanaFormat" change="updated()"></editor-checkbox>
```

Comment:
I do not think this is the right approach, can't have a bunch of options for different formats everywhere. Needs to be less intrusive / generic.